internal/vulnstore/postgres/querybuilder.go

```diff
@@ -24,58 +24,39 @@ func getBuilder(matchers []driver.MatchExp) (string, []driver.MatchExp, error) {
 	seen := make(map[driver.MatchExp]struct{})
 	deduped := []driver.MatchExp{}
 	for _, m := range matchers {
+		if _, ok := seen[m]; ok {
+			continue
+		}
+		var ex goqu.Ex
 		switch m {
+		case driver.PackageName: // ???
```

Comment:
Can we just nix these from this switch and from MatchExps ?

Reply:
Yeah, I think removing them and renaming those fields is in order. Is that a separate PR?

Reply:
@hdonnay you can make a judgement call on that. If it's a far-reaching rename and touches a lot maybe we punt to a dedicated PR. If it seems quick then by all means do it in this one.